customer-account-extension/locales/en.default.json

```diff
@@ -1,3 +1,4 @@
 {
-  "welcome": "Welcome to the {{target}} target of your customer account UI extension!"
+  "welcome": "Welcome to {{target}} target of your customer account UI extension ^_^",
+  "menuItemButton": "Open full page extension"
```

Comment:
Can we align with the UX guidelines here and keep this copy short so that it fits in one line? Both this and the French copy make the default thing partners see look pretty bad. 

<img width="321" alt="image" src="https://github.com/Shopify/extensions-templates/assets/17203119/c38b86f4-43a7-4cf0-b256-1fa68afa871d">